### PR TITLE
Add io.github.kukuruzka165.materialgram

### PR DESCRIPTION
**Pull request by incycledream**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link: https://github.com/kukuruzka165/materialgram/commit/b9636dd5830f4789d702d44c029c3ed5f2927bee**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: io.github.materialgram.yml [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+      - -Dvalidation=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libsigc++/3.6/libsigc++-3.6.0.tar.xz
+        sha256: c3d23b37dfd6e39f2e09f091b77b1541fbfa17c4f0b6bf5c89baef7229080e17
+        x-checker-data:
+          type: gnome
+          name: libsigc++
+
+  - name: glib
```
Comment (hfiguiere) [inline io.github.materialgram.yml:331]: this is in the runtime.

File: io.github.materialgram.yml [hunk 2]
```diff
@@ -0,0 +1,553 @@
...
+    config-opts:
+      - --disable-static
+      - --disable-examples
+      - --disable-doc
+    sources:
+      - type: git
+        url: https://gitlab.xiph.org/xiph/rnnoise.git
+        commit: 7f449bf8bd3b933891d12c30112268c4090e4d59
+
+  - name: qtbase
```
Comment (hfiguiere) [inline io.github.materialgram.yml:392]: you should use the KDE runtime then

File: io.github.materialgram.yml [hunk 3]
```diff
@@ -0,0 +1,553 @@
...
+        url: https://github.com/kukuruzka165/materialgram.git
+        commit: b9636dd5830f4789d702d44c029c3ed5f2927bee
+        tag: v4.11.1.1
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/kukuruzka165/materialgram/releases
+          tag-query: first | .tag_name
+          timestamp-query: first | .published_at
+          version-query: $tag | sub("^[vV]"; "")
+      - tdesktop.json
```
Comment (hfiguiere) [inline io.github.materialgram.yml:553]: why is this a separate manifest?

File: io.github.materialgram.yml [hunk 4]
```diff
@@ -0,0 +1,553 @@
+app-id: io.github.materialgram
```
Comment (hfiguiere) [inline io.github.materialgram.yml:1]: this app-id in invalid.

https://docs.flatpak.org/en/latest/conventions.html#application-ids

File: io.github.materialgram.yml (hunk 2 above)
Comment (incycledream) [reply io.github.materialgram.yml:392]: do I need to add only kde runtime or I need to have both kde runtime and freedesktop runtime?

**Comment by incycledream** [comment]
bot, build io.github.kukuruzka165.materialgram

**Comment by incycledream** [comment]
bot, build io.github.kukuruzka165.materialgram

**Comment by incycledream** [comment]
Works flawlessly on Fedora 40 (Rawhide) x86_64 with KDE Plasma 6.0 Beta 1
Screenshot_20231207_163550
Getting an error when trying to run from start menu
Screenshot_20231207_163041

**Comment by incycledream** [comment]
@hfiguiere will you accept my submission or I need to apply your changes?
I asked people about changes you suggested and they said it breaks my telegram client. also, following changes doesn't exist in other telegram clients (telegram desktop, 64gram)

**Comment by barthalion** [comment]
Doh.

**Comment by barthalion** [comment]
bot, build io.github.kukuruzka165.materialgram

**Pull request closed without merging**